data/DLAB/projects.yaml

```diff
@@ -94,3 +94,34 @@ projects:
             url: https://2021.emnlp.org/
     date_added: 2021-11-05
     date_updated: 2022-07-04
+
+  quotebank:
+    name: Quotebank
+    categories:
+      - Learning
+    applications:
+      - Info
+    description: Corpus of quotations from a decade of news
+    layman_desc: >
+      News from half a million of website, over the last 15 years, labelled by writer.
```

Comment:
Add something about how it can be used and where it's used.

Reply:
indeed a bit short; updated